examples/functional-iteration/functional_iteration.bal

```diff
@@ -44,7 +45,7 @@ public function main() {
 
     io:println("\nExecution Order:-");
     // The `forEach()` function applies the given function to each item of the container.
-    map<json> j = { name: "apple", colors: ["red", "green"], price: 5 };
+    map<json> j = {name: "apple", colors: ["red", "green"], price: 5};
```

Comment:
The formatting guide seems to have this approach too, but IMO with spaces looks cleaner. We may also have to update other BBEs too.

Reply:
Yes. This is according to the style guide and even our IDE tools formats according to that format.. We need to update all BBEs soon.